autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -111,11 +117,17 @@ def parse_instructor(course_data):
         email = faculty_data['emailAddress']
 
         instructor_model = Instructor(id=name, email_address=email)
-        instructor_model.save()
 
-        return instructor_model
+        if name not in instructors_set:
+            instructors_set.add(name)
+            return instructor_model
 
-def parse_course(course_data):
+        return None
```

Comment:
Delete this, every python function implicitly returns None when it reaches the end

Reply:
Fixed, squashed into "updated scrape_courses to use bulk_create"